### DISCUSSION TITLE
FPS going down

### DISCUSSION THREAD

**Discussion by KernelTech11** [ppy/osu#30251] [Q&A] [answered]
Why is my fps going down on osu!lazer? (ignore the sounds in the background) btw when i go select a song it gets worse

https://github.com/user-attachments/assets/6ae2a6d9-e11e-4eb9-b614-e94df9771675

Operational System: Windows 11 Insider Preview
Installed ram: 4gb
Processor: Intel Celeron CPU 4205U @ 1.80GHz
System Type: 64 bits

**Comment by shinmorisawa** [comment]
ooh... it might be a cpu bottleneck. osu! (especially osu!lazer) isn't meant to be run on cpus that underpowered. maybe upgrade time??? (you don't have to upgrade)

**Reply by KernelTech11** [reply, marked as answer]
i changed my renderer from OpenGL to Direct3D 11 and now it's performing as i would expect
